packages/neuron-wallet/src/services/node.ts

```diff
@@ -55,4 +65,6 @@ class NodeService {
   }
 }
 
+export const nodeService = NodeService.getInstance()
```

Comment:
If is it better to use `getter`, and use its instance via `NodeService.instance`

Reply:
any advice for variable name

Reply:
> any advice for variable name

No idea except for `ins` or `instance`

Reply:
As long as we use the same pattern, and have a consistent naming convention. `XxxService.getInstance()` looks just fine.

Reply:
reset it to `getter` until we know how to name variables .....

Reply:
That's fine, but my point is to reference the instance via `NodeService.getInstance()` instead of export one.

But they are almost the same, feel free to merge.

Reply:
I see. I think @classicalliu was looking at `startup/file-service` and did the same export here. Let's change all to `XxxService.getInstance()`.